tests/api/test_account.py

```diff
@@ -1258,9 +1258,7 @@ def test_password_reset_email(
     """
     email = customer_user.email
     variables = {"email": email}
-    response = staff_api_client.post_graphql(
-        query, variables, permissions=[permission_manage_users]
-    )
+    response = staff_api_client.post_graphql(query, variables)
```

Comment:
Why did we drop the permission checks?

Reply:
We have 2 mutations for resetting password.  The only difference between them is permission. I remove the permission form `PasswordReset` and set `CustomerPasswordReset` as deprecated.

Reply:
Then, shouldn't we use `customer_api_client` instead of `staff_client`?
Same thing for `test_password_reset_email_non_existing_user`